ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -25,25 +72,17 @@ public final class Atomic<Value> {
 	}
 	
 	/// Initializes the variable with the given initial value.
-	public init(_ value: Value) {
+	public init(_ value: Value, mutex: MutexType = PosixThreadMutex()) {
```

Comment:
I agree with @NachoSoto. I don't think this API should be public as-is. Let's keep the public API for `Atomic` the same.